Cargo.toml

```
@@ -16,6 +16,9 @@ include = [
     "README.md"
 ]
 
+# matches arrow 
+rust-version = "1.62"
```

```suggestion
rust-version = "1.70.0"
```

https://github.com/apache/arrow-rs/blob/a0148ba4cea1756bd22c534dff14e362cb5ee242/arrow/Cargo.toml#L34